controllers/memberoperatorconfig/configuration.go

```diff
@@ -171,6 +171,10 @@ func (a CheConfig) RouteName() string {
 	return commonconfig.GetString(a.che.RouteName, "codeready")
 }
 
+func (a CheConfig) IsDevSpacesMode() bool {
+	return a.Namespace() == "crw" && a.RouteName() == "devspaces"
```

Comment:
I would recommend checking the existence of the `codeready-workspaces-operator` in the first place. Basically, if the managed addon namespace still exists that would mean that CRW was not fully uninstalled from the cluster via OCM according to the migration SOP.

Reply:
Since the migration is a one time thing maybe we can just confirm the managed addon has been removed and everything is ready before changing the sandbox configuration?

Reply:
@ibuziuk don't forget that the `codeready-workspaces-operator` got stuck in the `terminating` state in the stage cluster. So checking that namespace wouldn't help at all.